packages/esm-patient-clinical-view-app/src/clinical-view-overview/clinical-view-overview.component.tsx

```diff
@@ -0,0 +1,106 @@
+import React from "react";
+import Button from "carbon-components-react/es/components/Button";
+import Tab from "carbon-components-react/es/components/Tab";
+import Tabs from "carbon-components-react/es/components/Tabs";
+import { useTranslation } from "react-i18next";
+import {
+  attach,
+  ExtensionSlot,
+  useConfig,
+  useCurrentPatient,
```

Comment:
Unused, or? (Just a minor thing; I am going to approve now - feel free to either remove unused parts or to merge)

Reply:
👍